README.md

```diff
@@ -98,6 +98,8 @@ Additional detailed documentation is included on the wiki on GitHub.
 
  * Added xSPUserProfileSyncConnection
 
+ * Added xSPFarmSolution resource
```

Comment:
You mean in the Header of the mof file, right? Haven't found anoterh place where the resources are documented.